### PR TITLE
Move MyApp to seperate class

### PR DESCRIPTION
**Pull request by Lee-Yoonjoo**
I moved MyApp class from Main to seperate class.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -16,36 +17,6 @@ void main() {
 }
 
 
-class MyApp extends StatelessWidget {
-  final List<ListItem> items;
-
-  const MyApp({Key? key, required this.items}) : super(key: key);
-
-  @override
-  Widget build(BuildContext context) {
```
Comment (mkoechig) [inline lib/main.dart:25]: Now I can review and comments directly in the changes

File: lib/screens/home.dart [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    return MaterialApp(
+      title: title,
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text(title),
+        ),
+        body: ListView.builder(
+          itemCount: items.length,
+          itemBuilder: (context, index) {
```
Comment (mkoechig) [inline lib/screens/home.dart:23]: I can even highlight multiple lines

File: lib/screens/home.dart [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            final item = items[index];
+
+            return ListTile(
+              visualDensity: VisualDensity(horizontal: 0, vertical: -4),
+              title:item.buildBody(context),
+            );
+          },
+        ),
```
Comment (mkoechig) [inline lib/screens/home.dart:31]: And you can answer to them or resolve them. Once all my comments are answered sufficiently, I can approve the pull request and you can merge. It's a good way to have a productive feedback loop.

**Review by mkoechig — CHANGES_REQUESTED** [review]

File: lib/screens/home.dart (hunk 3 above)
Comment (Lee-Yoonjoo) [reply lib/screens/home.dart:31]: Okay. Line by line confirmed

**Review by mkoechig — APPROVED** [review]

File: lib/screens/home.dart (hunk 2 above)
Comment (Lee-Yoonjoo) [reply lib/screens/home.dart:23]: Does everyone has permission to approve?

**Pull request merged**